notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -32,10 +32,10 @@
         <table role="presentation" style="border-collapse: collapse; max-width: 326px; width: calc(100% - 10px) !important;" align="left">
           <tr>
             <td style="padding-left: 10px;">
-              
+
               {% if fip_banner_english %}
               <img
-                src="https://notification-alpha-canada-ca-asset-upload.s3.amazonaws.com/gov-canada-en-01.png"
+                src="https://assets.notification.canada.ca/gov-canada-en.png"
```

Comment:
There was already a file without `-01.png` in the bucket. I confirmed that these files were the same and used the cleaner filename